### PR TITLE
Add: Retrained 2020 and 2021 models

### PR DESCRIPTION
**Pull request by bhyeh**
Retrained Ethiopia models to address and fix two bugs:
1. 2020 model had a 'lookahead' leak where it had been mistakenly trained on _2021_ data as well
2. 2021 model had _not_ been trained w/o ERA5 data

Also address bug in `train.py` w/ building the default training dataset that was introduced w/ adding Ethiopia Tigray ground truth to .dvc repository.

### CODE REVIEW

**Comment by bhyeh** [comment]
Missing the changed `models.dvc` file - changes to DVC are being blocked/stopped by Windows Virus Protection.

File: data/models.dvc [hunk 1]
```diff
@@ -1,5 +1,5 @@
 outs:
-- md5: b5eec55168aac3c37ebb280278aba1d3.dir
-  size: 39869012
-  nfiles: 28
+- md5: 5a2212cd3f931d028a95ec55b6ec2f7b.dir
+  size: 30209284
+  nfiles: 22
```
Comment (bhyeh) [inline data/models.dvc:4]: Added changed `models.dvc` - is this nfile change correct? Were models removed from the dvc repository recently?

Comment (ivanzvonkov) [reply data/models.dvc:4]: `models.dvc` inside the `master` branch is 28 so I think this branch is the only one where the change is occurring. I recommend pulling the two model files to a separate location and doing a `dvc pull` using the prior `models.dvc` file.

**Review by ivanzvonkov — APPROVED** [review]
- Forced model.dvc back to original version
- Took opportunity to retrain models with your train Github action!

> 1. 2020 model had a 'lookahead' leak where it had been mistakenly trained on 2021 data as well
I don't see an issue with this since the points are not the same from 2020 and 2021
image

> 2. 2021 model had not been trained w/o ERA5 data
Updated the Github action so both trained models skip_era5

> Also address bug in train.py w/ building the default training dataset that was introduced w/ adding Ethiopia Tigray ground truth to .dvc repository.

It was skipped before because the start_date was January and we have an unenforced requirement that datasets should start in January so that they can be concatenated. I missed this when you added the dataset and addressed it here #286 

Let me know if you have any questions about my updates, otherwise I am good with merging this.

**Comment by bhyeh** [comment]
Looking good! Thanks for the help here, Ivan. I am only curious about how you were able to train w/ the GH action and specify `skip_era5`? Otherwise I will merge this.

**Comment by ivanzvonkov** [comment]
I added an optional model args parameter where you can put whatever argument you want: https://github.com/nasaharvest/crop-mask/blob/8fe87ec10a0455e48f9fdd6ea75b630034411b7f/.github/workflows/train.yml#L18
